translib/app_utils.go

```diff
@@ -68,19 +70,16 @@ func getYangPathFromYgotStruct(s ygot.GoStruct, yangPathPrefix string, appModule
 	return ""
 }
 
-func generateGetResponse(targetUri string, root *ygot.GoStruct, fmtType TranslibFmtType) (GetResponse, error) {
+func generateGetResponsePayload(targetUri string, deviceObj *ocbinds.Device, ygotTarget *interface{}, fmtType TranslibFmtType) ([]byte, ygot.ValidatedGoStruct, error) {
```

Comment:
@mbalachandar, can you revert changes to this API? I had cleaned it up earlier for simpler usage. Instead of every app receiving 3 return values and creating a GetResponse object, it is better to prepare the GetResponse object from here itself. Also the parameter "ygotTaregt" is never used inside this function. Hence I had removed it. I think can revert this API change and all corresponding app changes.